roles/gateway/README.rst

```diff
@@ -58,6 +61,63 @@ A list of users that don't have their ssh pubkey added to the ``teuthology_user`
     openvpn_users:
       - ovpn: user@host etc...
 
+The following vars are used to populate ``/etc/resolv.conf``.  Defined in the
+secrets repo::
+
+    gw_resolv_search: []
+    # Example: gw_resolv_search: "front.example.com"
+
+    gw_resolv_ns: []
+    # Example:
+    gw_resolv_ns:
+      - 1.2.3.4
+      - 8.8.8.8
+
+The ``gw_networks`` var assumes you have individual "physical" NICs for each
```

Comment:
just so I understand: they're not really physical because you're using a vm, but that illusion is opaque to the code, and this would work on a physical host where they really are physical, right?...